cli/cmd/install.go

```diff
@@ -472,8 +472,12 @@ func doInstallation() error {
 	}
 
 	logging.PrintLog("Keptn has been successfully set up on your cluster.", logging.InfoLevel)
-	logging.PrintLog("To connect the Keptn CLI with the Keptn API on your cluster, "+
-		"please refer to the instructions at: https://keptn.sh/docs/"+keptnReleaseDocsURL+"/operate/install", logging.InfoLevel)
+	logging.PrintLog("---------------------------------------------------", logging.InfoLevel)
```

Comment:
For user messages, please use `fmt.Println`

Reply:
I changed it to `fmt.Println`